app/helpers/application_helper/button/ph_infra_console.rb

```diff
@@ -0,0 +1,13 @@
+class ApplicationHelper::Button::PhInfraConsole < ApplicationHelper::Button::Basic
+  needs(:@record)
+
+  def visible?
+    @record.supports_console?
+  end
+
+  private
+
+  def ems?
```

Comment:
Why a `ems?` returns an id instead of a boolean? Question marks are used to define a boolean method, in ruby convention.